wikimedia/wikipedia-ios
# T378507 UI fixes

**Phabricator:** 
https://phabricator.wikimedia.org/T378507#10301445

### Notes
* Added icon in intro slide
* Updated toolbar appearance to avoid pull down color issue

### Test Steps
1. In English, Spanish, or Italian in Italy or Mexico in scheme
2. Open app, log in if not logged in
3. Enable year in review feature flag in developer settings
4. Navigate through 6+ articles, background and foreground app.
5. Navigate to Year in Review
6. Ensure icon is in intro slide.
7. Navigate to one of the other slides, and try a pull down as far as possible, ensure background of bottom toolbar remains the proper color.
8. Repeat with other background options. 

### Screenshots/Videos

https://github.com/user-attachments/assets/ecfbffe9-3be5-46ea-a64b-031ba26f47b3
